conduct.md

```diff
@@ -212,4 +232,4 @@ Any decision made by the instructor for the given infraction(s) will be final.
 ## Names of Individuals who Agree to Adhere to the Guide
 
 Janyl Jumadinova
-:heavy_exclamation_mark: TODO: Add your name
+Nate Angus
```

Comment:
Please make sure to add the names of all group members. Additionally, the `TODO` section should be kept until other groups have also added their names